make_ghpages/mod/templates/base.html

```diff
@@ -38,9 +39,16 @@ <h1>
     {% block footer %}
     <footer class="footer">
     <hr>
-    This page is part of the <a href="{{ app_registry_url }}">BIG-MAP registry</a>, the official registry of <a href="{{ big_map_url }}">BIG-MAP</a> applications.
-    <br>
-    The BG-MAP project has received funding from the European Union’s <a href="https://ec.europa.eu/programmes/horizon2020/en" target="_blank">Horizon 2020 research and innovation programme</a> under grant agreement <a href="https://cordis.europa.eu/project/id/957189" target="_blank">No 957189</a>.
+    <div class="side-by-side">
+      <div>
+        <img src="{% block eulogoimagesrc %}static/img/eu-logo.png{% endblock %}" style="width: 60px; vertical-align: middle;" width="60px">
+      </div>
+    <div style="width: 80%;">
+      This page is part of the <a href="{{ app_registry_url }}">BIG-MAP registry</a>, the official registry of <a href="{{ big_map_url }}">BIG-MAP</a> applications.
+      <br>
+      The BIG-MAP project has received funding from the European Union’s <a href="https://ec.europa.eu/programmes/horizon2020/en" target="_blank">Horizon 2020 research and innovation programme</a> under grant agreement <a href="https://cordis.europa.eu/project/id/957189" target="_blank">No 957189</a>.
+      </div>
```

Comment:
This block is not correctly intended. It's inconsequential, but maybe we can fix that quickly?

Reply:
I admittedly have issues with HTML indentation ;-)
I decided to reindent the whole file with VSCode, hope that's OK (I guess the only approach would be to have some pre-commit to standardise this, but I don't think it's worth it for this repo).